### DISCUSSION TITLE
Rolladen Automatisation Fehler

### DISCUSSION THREAD

**Discussion by Varone3000** [brutella/hkknx-public#236] [Q&A] [answered]
Moin Matthias,

ich nutze die Testvariante und bin auch im KNX Forum unter Varone3000 zu finden.
Folgender Sachverhalt, ich habe die Rollladen aus dem KNX in HKKNX eingebunden und ich kann sie auch fahren. Allerdings fährt die Routine die Rollos Abends nicht zu. Wenn ich allerdings sage, teste Automatisation klappt aber alles.
Irgendeine Idee? Desweiteren wird Auf der Hauptseite von Homekit der Rolladen als Klima angezeigt.

**Comment by chef76** [comment, marked as answer]
Ist das eine zeitgesteuerte Automation in HomeKit? Dann hast du vermutlich die Zeit nach dem Anlegen der Automation noch mal geändert (aktueller Bug in HomeKit). Lösch die Automation und leg sie mit der gewünschten Zeit neu an, dann gehts!

**Reply by Varone3000** [reply]
Moin,
Ja es war tatsächlich so.
Was aber interessant dabei ist, er scheint den Rolladenaktor ganz kurz anzusprechen, den er klackt kurz.
Ist das also ein Apple Fehler und kein HKNX Problem?

**Reply by chef76** [reply]
Ja, da kann HKKNX nichts dafür. Mir ist es zuerst bei HomeKit Geräten aufgefallen.
